docdoku-common/src/main/java/com/docdoku/core/services/IImporterManagerLocal.java

```diff
@@ -34,7 +34,7 @@
 
     Future<ImportResult> importIntoParts(String workspaceId, File file, String originalFileName, String revisionNote, boolean autoCheckout, boolean autoCheckin, boolean permissiveUpdate) throws Exception;
     Future<ImportResult>  importIntoPathData(String workspaceId, File file, String originalFileName, String revisionNote, boolean autoFreezeAfterUpdate, boolean permissiveUpdate) throws Exception;
-    ImportPreview getCheckoutPartToImport(String workspaceId, File file, String originalFileName) throws Exception;
+    ImportPreview dryRunCheckout(String workspaceId, File file, String originalFileName) throws Exception;
```

Comment:
It's ok that we report the list of parts that will be checked out but the operation that we dry run is the import operation.
Thus rename it to dryRunImportIntoParts and add the following parameters (boolean autoCheckout, boolean autoCheckin, boolean permissiveUpdate) so we can accurately mimic to import operation.